utils/build-script-impl

```diff
@@ -2329,7 +2330,15 @@ for host in "${ALL_HOSTS[@]}"; do
                         -DSWIFT_TENSORFLOW_TARGET_INCLUDE_DIR:PATH="${TENSORFLOW_TARGET_INCLUDE_DIR}"
                     )
 
-                    # Handle TensorFlow Swfit bindings file.
+                    # Propagate TensorFlow GPU support flag to CMake.
+                    if [[ "${ENABLE_TENSORFLOW_GPU}" ]] ; then
+                        cmake_options=(
+                            "${cmake_options[@]}"
+                            -DSWIFT_ENABLE_TENSORFLOW_GPU:BOOL=TRUE
```

Comment:
why do we need this cmake option?

Reply:
It's not currently used, but it could be useful in CMake for gating behavior.

I originally added it so that `-DCPU` and `-DCUDA` would both be enabled when `SWIFT_ENABLE_TENSORFLOW_GPU` is true.

Reply:
SG. would you like to add a comment that this is not currently used?

Reply:
Done in https://github.com/apple/swift/pull/17839/commits/71386d2bd4e23db27357142389d120d67fb7a5b4.